app/helpers/items_helper.rb

```diff
@@ -1,11 +1,8 @@
 module ItemsHelper
   def bought_by item, user
-    orders = user.orders
-    line_items = orders.map { |order| order.line_item }
-    items = line_items.map { |line_item| line_item.item }
-    if items.include? self
-      return true
-    end
-    return false
+    return false unless user
+    items = user.orders.where(status: 'completed').
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.